src/d3a/d3a_core/export.py

```diff
@@ -450,6 +503,43 @@ def __init__(self, dataset: dict, key: str):
         self.key = key
         self.dataset = dataset
         self.umHours = dict()
+        self.rate = list()
+        self.energy = list()
+
+    def sd_curve(self, supply=True):
+        try:
+            if supply:
+                sort_values = SortedDict(self.dataset)
+                self.rate = list(sort_values.keys())
+                self.energy = list(sort_values.values())
+            if not supply:
+                sort_values = SortedDict(self.dataset)
+                self.rate = list(reversed(sort_values.keys()))
+                self.energy = list(reversed(sort_values.values()))
+
+            cond_rate = list()
+            cond_energy = list()
+
+            for i in range(len(self.energy)):
+
+                if i == 0:
+                    cond_rate.append(self.rate[0])
+                    cond_energy.append(0)
+                    cond_rate.append(self.rate[0])
+                    cond_energy.append(self.energy[i])
```

Comment:
As i is 0 here, this is a duplicate of line 527, right?

Reply:
Its not duplicate, Im trying to add two points in x-axis with same energy_rate. If it is not clear, I can draw and explain offline.